build.ps1

```diff
@@ -330,6 +330,18 @@ function Append-FlagsDefine([hashtable]$Defines, [string]$Name, [string]$Value)
   }
 }
 
+function Test-CMakeMinVersion([int]$Major, [int]$Minor, [int]$Patch = 0) {
```

Comment:
Does `Test-CMakeAtLeast` fit the naming scheme for PS?  If so, that seems like a nicer name.

Reply:
I don't know if it's legit but I took your suggestion.